flutter_local_notifications/README.md

```diff
@@ -337,6 +337,12 @@ Developers should also be across Google's requirements on using full-screen inte
 
 ⚠️ Ensure that you have configured the resources that should be kept so that resources like your notification icons aren't discarded by the R8 compiler by following the instructions [here](https://developer.android.com/studio/build/shrink-code#keep-resources). If you have chosen to use `@mipmap/ic_launcher` as the notification icon (against the official Android guidance), be sure to include this in the `keep.xml` file. If you fail to do this, notifications might be broken. In the worst case they will never show, instead silently failing when the system looks for a resource that has been removed. If they do still show, you might not see the icon you specified. The configuration used by the example app can be found [here](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/example/android/app/src/main/res/raw/keep.xml) where it is specifying that all drawable resources should be kept, as well as the file used to play a custom notification sound (sound file is located [here](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/example/android/app/src/main/res/raw/slow_spring_board.mp3)).
 
+#### proguard-rules.pro
```

Comment:
Since another (sub)section has been added. Could you update the TOCs so there's a link that allows jumping to this one? I would also suggest renaming this to `ProGuard rules`

Reply:
Thanks! I fixed spell and added link. [60086c2](https://github.com/MaikuB/flutter_local_notifications/pull/2552/commits/60086c28705ad6b85874a43c6197e84284ff1ccb)